doc/Language/objects.pod6

```diff
@@ -286,9 +283,9 @@ attributes in initializers.
 
 =head2 Private Methods
 
-Methods that are not going to be called or callable from anywhere outside
+Methods that are not going to be called or are not callable from anywhere outside
```

Comment:
We shouldn't say “Methods that are not going to be called”. Should begin with: “Methods with the exclamation mark…”. The the next sentence should, and leave out the part about the objects not being called. Leave out the “not going to be called” part, since it's the methods that are private, they aren't private just because they are never called.